roles/nvidia-hpc-sdk/tasks/main.yml

```diff
@@ -30,6 +30,7 @@
     url: "{{ hpcsdk_download_url }}"
     dest: "{{ hpcsdk_dest_download_path }}"
     mode: "0444"
+    validate_certs: no
```

Comment:
What problem did you run into here? It's generally a bad idea to not validate certs, and the HPC SDK download site should have valid certs.

Reply:
I was helping a client run these scripts and on their system it refused to download the HPC SDK until the validate certs setting was set to no. I can remove this.

Reply:
Yeah, that sounds like a problem with their system configuration. In general it's a bad idea to not validate certs. 😄

Reply:
Then I'll remove this.

Reply:
Fixed in commit 42a8792.